resources/views/components/backend/taxonomy_property_adder.blade.php

```diff
@@ -0,0 +1,186 @@
+        <button type="button" class="btn btn-primary btn-w-150" data-bs-toggle="modal" data-bs-target="#addPropertyItemModal"
+            x-on:click="resetForm(); document.getElementById('addPropertyItemModalLabel').innerHTML = 'Add Property'; isEditing=false">
+            <i class="fas fa-plus me-2"></i>Add
+        </button>
+        <button type="button" class="btn btn-dark btn-w-150" x-show="properties.length" x-transition x-on:click="ClearAll()">
+            <i class="fas fa-times me-2"></i>Clear All
+        </button>
+    </div>
+    
+    <!-- Modal -->
+    <div class="modal fade" id="addPropertyItemModal" tabindex="-1" aria-labelledby="addPropertyItemModalLabel"
+        aria-hidden="true" x-init="$el.addEventListener('hidden.bs.modal', () => resetForm())">
+        <div class="modal-dialog modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h1 class="modal-title fs-5" id="addPropertyItemModalLabel">Add Property</h1>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <form >
+                    <div class="modal-body">
+                        <div class="container-fluid">
+                            <div class="row">
+                                <div class="mb-3">
+                                    <label for="propertyCode" class="form-label">Code*</label>
+                                    <input type="text" class="form-control" id="propertyCode" 
+                                        x-model="newProperty.code" />
+                                </div>
+                                <div class="mb-3">
+                                    <label for="propertyName" class="form-label">Name*</label>
+                                    <input type="text" class="form-control" id="propertyName" 
+                                        x-model="newProperty.name"/>
+                                </div>
+                                <div class="mb-3">
+                                    <label for="propertyDatatype" class="form-label">Datatype</label>
+                                    <select class="form-select" aria-label="Datatype" x-model="newProperty.data_type">
+                                        <option selected value="string">String</option>
+                                        <option value="integer">Integer Number</option>
+                                        <option value="float">Floating Point Number</option>
+                                        <option value="date">Date</option>
+                                        <option value="datetime">Date Time</option>
+                                        <option value="boolean">Boolean</option>
+                                        <option value="url">URL</option>
+                                        <option value="image">Image</option>
```

Comment:
Can we move this list into the Taxonomy Model level as a key-value dictionary named `propertyType`? (Then it is easy to add new types in future)